lessons/advanced/protocols.md

```diff
@@ -110,6 +110,6 @@ iex> to_atom(%{foo: "bar"})
 :foo
 ```
 
-It is worth nothing that although underneath structs are Maps, but they do not share protocol implementations with Maps. They are not enumerable, they cannot be accessed.
+It is worth noting that although underneath structs are Maps, but they do not share protocol implementations with Maps. They are not enumerable, they cannot be accessed.
```

Comment:
I didn't have issues as well, I just thought that it was a typo, hence the issue. Both of your suggestions seem better to me than the original.